deploy/jenkins-ci/Jenkinsfile

```diff
@@ -140,6 +140,18 @@ node('kiali-build') {
           }
         }
       }
+
+     stage('Post Msg Bus Notification') {
+       when { environment name: 'RELEASE_TYPE', value: 'minor'}
+         steps {
+           build(job: 'kiali-release-notifier',
+             parameters: [
+               [$class: 'StringParameterValue', value: "${params.RELEASE_TYPE}", name: 'RELEASE_TYPE'],
```

Comment:
The `RELEASE_TYPE` parameter, most of the time, will have `auto` value. Please, use `${releaseType}` variable which will have the resolved value.